R/read_from_fingertips.R

```diff
@@ -20,6 +20,9 @@
 #' @return a `list` of 1 element of type `data.frame`. This contains the
 #'    imported dataset of interest.
 #'
+#' @keywords internal
+#' @noRd
```

Comment:
```suggestion
```

Let's try removing this for now and then we'll adjust based on CRAN's feedback in the future. The policy for this has been slightly inconsistent.